Source/VideoStreaming/igtlVideoMessage.cxx

```diff
@@ -70,7 +70,8 @@ namespace igtl {
     scalarType    = TYPE_UINT8;
     m_FrameHeader = NULL;
     m_Frame       = NULL;
-    
+    coordinate = COORDINATE_RAS;
```

Comment:
So, this is applying a medical imaging convention to a generic video message. I don't know if this is wise. @lassoan ?

Reply:
If it's being used as a right-handed/left-handed indicator, please rename accordingly.

Reply:
When I was copying over the transformation information from the image message, I copied the coordinate definition over assuming that it could be useful.

In retrospect, it does't make a whole lot of sense.

Should be easy to remove in any case, if that's the case.

Reply:
RAS and LPS are both right-handed. It is important to specify coordinate system. We could specify that anatomical coordinate system is always LPS in OpenIGTLink, but since it was not a requirement for IMAGE message, it would not decrease complexity of compliant software, it would just make this particular message type less flexible.